blog/let's-roll.mdx

```diff
@@ -0,0 +1,59 @@
+
+## TL;DR:
+
+-Rollkit is launching its community call, Let’s Roll. You can subscribe to the community calendar [here](https://calendar.google.com/calendar/u/0/embed?src=c_11a8d6a470ca9c07b131ab1ea213ac124a3061a9d6986bb208d88afbcc4bb3d2@group.calendar.google.com&ctz=America/Toronto)
+
+-Builders using Rollkit can now present their projects at Let’s Roll. Reach out to us on [Telegram](https://t.me/rollkit) 
+
+## Just merge it
+
+Rollup development is rapidly evolving, with new types of rollups emerging, like sovereign and pessimistic rollups, and with new possibilities of using layer 1 blockchains as data availability layers. Rollkit is at the forefront of the rollup movement offering an open source, public good and modular framework for rollups. 
+
+However, Rollkit's true potential lies in its community of developers and users who bring their unique perspectives, skills, and expertise to the table. When Rollkit integrates a new data availability layer, its community members are best suited to improve this integration and use it in ways that truly benefit the community. Therefore, we invite developers from different communities and skill sets to join us and contribute to Rollkit’s development.
+
+By contributing to Rollkit, we reduce fragmentation, duplicated features and bugs, and we ensure the continuous improvement of Rollkit. By doing that we forge a stronger and bigger community. 
+
+One framework is bigger than the sum of all its forks. Together we can build something for all developers and builders to easily use.
+
+Our promise, as core contributors, is to do our best to make “just merge it” a reality. We strive to empower developers to contribute to Rollkit and ensure that their contributions are integrated. 
+
+That's why we believe it's time to bring our community together to share knowledge, collaborate on new ideas, and inspire each other to push the limits of what's possible with Rollkit.
+
+## What is Let’s Roll?
+
+Let's Roll is a dynamic community-focused meeting that brings together all Rollkit’s contributors to drive the innovation and development of Rollkit.
+
+This meeting serves as a platform for sharing knowledge, discussing the latest updates, and exploring new ideas. Rollkit's contributors will demonstrate the most recent features and advancements, while community members will have the chance to ask questions, provide feedback, and collaborate on new projects.
+
+Building open source means encouraging collaboration and teamwork among all developers. Sharing code and ideas will help us build a better framework more efficiently.
+
+The meeting will be hosted on [Zoom](https://us06web.zoom.us/j/83308619122). Subscribe to the following [Google calendar](https://calendar.google.com/calendar/u/0/r?cid=c_11a8d6a470ca9c07b131ab1ea213ac124a3061a9d6986bb208d88afbcc4bb3d2@group.calendar.google.com) so that you can be notified of future community calls.
```

Comment:
```suggestion
Let's Roll is a dynamic community-focused meeting that brings together all of Rollkit’s contributors to drive the innovation and development of Rollkit.

This meeting is a platform for sharing knowledge, discussing the latest updates, and exploring new ideas. Rollkit's contributors will demonstrate the most recent features and advancements, while community members can ask questions, provide feedback, and collaborate on new projects.

Building open source means encouraging collaboration and teamwork among all developers. Sharing code and ideas will help us build a better framework more efficiently.

The meeting will be hosted on [Zoom](https://us06web.zoom.us/j/83308619122). In addition, subscribe to the following [Google calendar](https://calendar.google.com/calendar/u/0/r?cid=c_11a8d6a470ca9c07b131ab1ea213ac124a3061a9d6986bb208d88afbcc4bb3d2@group.calendar.google.com) so that you can be notified of future community calls.
```